lib/Styles/Core.less

```diff
@@ -223,6 +224,14 @@ button.block{
   h5 { font-size: .875rem }
   h6 { font-size: .75rem }
 
+  td {
+    min-width: 100px;
```

Comment:
This line makes me a little nervous.  There are legitimate reasons to have a narrow column, right?  If we leave this off, the "Related Maps" image will be really small..  but that's ok, because we've already added a `width` to that `<td>` to fix that anyway.  So are you ok with just getting rid of this `min-width`?